resources/resincli.robot

```diff
@@ -151,6 +155,17 @@ Check if "${option}" variable "${variable_name}" with value "${variable_value}"
     ${result} =  Run Process    echo "${result_env.stdout}" | grep ${variable_name} | grep " ${variable_value}"    shell=yes
     Process ${result}
 
+Check if "${option}" variable "${variable_name}" with value "${variable_value}" exists on "${type}" "${id}"
+    [Documentation]    Available values for argument ${option} are: ENV, CONFIG
+    @{list} =  Create List    ENV    CONFIG
+    Should Contain    ${list}    ${option}
+    ${result_env} =  Run Keyword If    '${option}' == 'ENV'    Run Process    balena envs --${type} ${id} | sed '/ID[[:space:]]*NAME[[:space:]]*VALUE/,$!d'    shell=yes
```

Comment:
What is the goal of this sed command?

Reply:
no idea, didn't write it and it seems to work, so not keep to touch.